src/en/standards/backgrounder.md

```diff
@@ -0,0 +1,468 @@
+layout: layouts/base.njk
+description: "<p>This document provides an overview of the requirements we plan to include in phase one of the <em><a href='../standard'>Standard on Information and Communication Technology Accessibility (<abbr>ICT</abbr>)</a></em>.</p>
+<p>It provides background information, explanations of each requirement and some publicly available resources.</p>"
+internalLinks: true
+---
+
+**NOTE:** It is important to note the information in this document is for reference purposes only and **must not be interpreted as being official standard guidance**. Official standard guidance will be published at a later date and will provide detailed information on how departments can satisfy each standard requirement.
+
+## Context
+
+One in six people in Canada has a disability. Accessible information and communication technology ensures that people with disabilities can access and use Government of Canada services without barriers. Making services accessible to people with disabilities makes them accessible to everyone.
+
+As technology continues to evolve, the Government of Canada has an obligation to make sure the next generation of technological solutions are developed with accessibility at their core so that no one is left behind.
+
+The release of the _[Accessible Canada Act](https://laws-lois.justice.gc.ca/eng/acts/A-0.6/)_, the [Accessibility Strategy for the Public Service of Canada](https://www.canada.ca/en/government/publicservice/wellness-inclusion-diversity-public-service/diversity-inclusion-public-service/accessibility-public-service/accessibility-strategy-public-service-toc/accessibility-strategy-public-service-technology.html) and the [Guideline for Making Information Technology Usable for All](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32620) in 2019 led to many initiatives across the Government of Canada to make <abbr title="Information and Communication Technology Accessibility">ICT</abbr> barrier-free. One of these initiatives was the drafting of the _Standard on Information and Communication Technology Accessibility_. This standard, which will replace the _Standard on Web Accessibility_, will expand the scope of accessibility requirements under the _Policy on Service and Digital_. The standard will be harmonized with the <a download="download" href="https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf"><em>Harmonised European Standard on Accessibility Requirements for <abbr title="Information and Communication Technology Accessibility">ICT</abbr> Products and Services (2021) (EN&nbsp;301&nbsp;549)</em> (PDF, 2.3MB)</a> to: meet all relevant requirements in the act and strategy; align with international best practices; and align with <a download="download" href="https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf"><em>EN&nbsp;301&nbsp;549</em> (PDF, 2.3MB)</a> (2021 and future updates).
```

Comment:
For the PDF link
```
<a download="download" href="https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf"><em>EN&nbsp;301&nbsp;549</em> (PDF, 2.3MB)</a>
```

Worth to consider to use the "type" attribute on it because that is a PDF. 

ref. https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-type
list of all mime type: https://www.iana.org/assignments/media-types/media-types.xhtml